build/images/scripts/start_ovs_windows_containerd.ps1

```diff
@@ -0,0 +1,171 @@
+$ErrorActionPreference = "Stop"
+$OVSVersion = "3.0.5-antrea.0"
```

Comment:
Can we use a parameter to provide the OVS version to avoid hard code?

Reply:
@NamanAg30 ,
ovs version is maintained in a common file, so can directly use the version from there, version info must not be hardcoded and use windows ovs version common file to access ovs version.

Reply:
ovs version parameter not being used now